src/feature/trips/components/modals.js

```diff
@@ -1,10 +1,14 @@
-import React from 'react'
-import "../styles/modals.scss";
-import { boardingPoints, droppingPoints } from "../data";
-import { formatDate, formatTime } from './formatDatetime';
+import React from "react";
+// import { boardingPoints, droppingPoints } from "../data";
+import { formatDate, formatTime } from "./formatDatetime";
+import { stopPointsSelector } from "../redux/selectors";
+import { useSelector } from "react-redux";
 import { aminitiesSvg } from "./svg";
+import "../styles/modals.scss";
 
-export function DroppingBoardingPoint () {
+export function DroppingBoardingPoint() {
+  const { boardingPoints, droppingPoints } = useSelector(stopPointsSelector);
```

Comment:
boarding & droppingPoints depends on tripId , make sure to pass `tripId` stopPointsSelector